app/controllers/healthcheck_controller.rb

```diff
@@ -0,0 +1,5 @@
+class HealthcheckController < ApplicationController
+  def index
+    render json: {}
```

Comment:
Not sure whether you think it makes sense to also check that the DB and queues are operational. If so, you could add this before the `render` call:

```rb
# Ensure database is available
ActiveRecord::Base.connection.exec_query('SELECT 2 + 2;')
# Ensure job queue is available
Resque.size(:not_a_real_queue_but_thats_ok)
```

Both will raise exceptions if they can’t connect.

Reply:
The `Rescue...` line errors out:

```
Error:
Api::V0::DiffControllerTest#test_healthcheck_responds_with_success:
Errno::EADDRNOTAVAIL: Cannot assign requested address - connect(2) for [::1]:6379
    app/controllers/healthcheck_controller.rb:6:in `index'
    test/controllers/healthcheck_controller_test.rb:7:in `block in <class:DiffControllerTest>'
```

Is that quite right?